webapi/src/test/scala/org/knora/webapi/messages/admin/responder/listsmessages/ListsMessagesADMSpec.scala

```diff
@@ -179,6 +180,39 @@ class ListsMessagesADMSpec extends AnyWordSpecLike with Matchers with ListADMJso
             thrown.getMessage should equal ("Invalid list IRI")
         }
 
+        "throw 'BadRequestException' for `CreateListApiRequestADM` when value of a label is missing" in {
+
+            val payload =
+                s"""
+                   |{
+                   |    "projectIri": "${SharedTestDataADM.IMAGES_PROJECT_IRI}",
+                   |    "labels": [{ "value": "Neuer List Node", "language": "de"}, { "value": "", "language": "en"}],
+                   |    "comments": []
+                   |}
+                """.stripMargin
+
+            val thrown = the [BadRequestException] thrownBy payload.parseJson.convertTo[CreateListApiRequestADM]
+
+            thrown.getMessage should equal ("String value is missing.")
+        }
+
+        "throw 'BadRequestException' for `CreateListApiRequestADM` when value of a comment is missing" in {
+
+            val payload =
+                s"""
+                   |{
+                   |    "parentNodeIri": "$exampleListIri",
+                   |    "projectIri": "${SharedTestDataADM.IMAGES_PROJECT_IRI}",
+                   |    "labels": [{ "value": "Neuer List Node", "language": "de"}],
+                   |    "comments": [{ "value": "", "language": "de"}]
+                   |}
+                """.stripMargin
+
+            val thrown = the [BadRequestException] thrownBy payload.parseJson.convertTo[CreateListApiRequestADM]
+
+            thrown.getMessage should equal ("String value is missing.")
```

Comment:
I'm generally reluctant to check the contents of exception messages in tests, because exception messages often change over time, and then it's annoying to have to fix a lot of tests. But it's up to you.

Reply:
@benjamingeer yeah, I need to check them. It is my paranoia ;-)